src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -119,20 +119,33 @@ class SpanDetail extends React.Component<Props, State> {
       );
     }
 
-    const {span, orgId} = this.props;
+    const {span, orgId, trace} = this.props;
+
+    const {start, end} = getTraceDateTimeRange({
+      start: trace.traceStartTimestamp,
+      end: trace.traceEndTimestamp,
+    });
 
     const eventView = EventView.fromSavedQuery({
       id: undefined,
       name: t('Transactions'),
-      fields: ['transaction', 'trace.span', 'timestamp'],
-      fieldnames: ['transaction', 'trace.span', 'timestamp'],
+      fields: [
```

Comment:
Would using a name like `Childen of {span.description} at {startTimestamp}` help communicate why/how the user ended up on these results?

Reply:
@doralchan what do you think?

Reply:
Dora doesn't like it.